src/wallet_client.ts

```diff
@@ -605,9 +605,7 @@ export class WalletClient {
         let objData = await this.provider.getObject(obj.objectId);
         let moveObj = getMoveObject(objData);
         if (
-          moveObj!.fields.name &&
-          moveObj!.fields.description &&
-          moveObj!.fields.url
+          !Coin.isCoin(objData)
```

Comment:
Can we instead make the check for url? From our call, it seems like we can certainly say that an object containing a url will be an nft

Reply:
Ok sure will do that